app-text/dblatex/Manifest

```diff
@@ -0,0 +1,5 @@
+AUX dblatex-0.3.7-install_layout.patch 502 SHA256 53ad2207b9f1181f16e7fe8bae919d5a74f4d22ba15945c61322a0fda06472f0 SHA512 e8ad310d83425f18f1711452a2c23d7159807a323faf3fc698c554548fac5dd1d1ac634ab13d7a7cd0f87fc4258ec0b833370424ec687df9cef290a802b41174 WHIRLPOOL 15fd2d116272adecb271533111a40469274005c3f9b72b05c95ed42cf739c95e62d8504509ee0836df9fc855ac9a6e2c311b648c478251a3159caff45776c6b7
```

Comment:
This is not a valid thin Manifest. Which indicates you copied it from some overlay without regenerating.